tests/snapshots/tests.contrib.django.test_django_snapshots.test_asgi_200_30.snap

```diff
@@ -1,22 +1,22 @@
-[[{"name" "asgi.request"
+[[{"name" "django.request"
    "service" nil
```

Comment:
Shouldn't the service have been set as we are passing in the integration config?

Reply:
Yeah I had to regenerate the snaps after the integration config change